tsconfig.json

```diff
@@ -19,12 +19,12 @@
 		"noUnusedParameters": false,
 		"noUnusedLocals": false,
 
-		"sourceMap": true
+		"sourceMap": true,
 
-	},
-
-	"files": [
```

Comment:
What I've found is that IntelliJ will find the tsconfig if it's in the root path, but it won't actually apply all of the settings to your ts files if you haven't identified (either using "files" or "include") which files it should apply to. There are even situations where tsc will complain if both the "files" or "include" list are empty.